_gtfobins/cpulimit.md

```diff
@@ -0,0 +1,11 @@
+---
+functions:
+  execute-interactive:
+    - description: With -f, the commands are executed in the terminal you start them. -l 100 specified that the command is to use a single CPU core
```

Comment:
I'd drop the description here, usually descriptions should be about gotchas, for the rest users can lookup the man.